docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -2,11 +2,21 @@
 SQL Server connector
 ====================
 
-The SQL Server connector allows querying and creating tables in an
-external `Microsoft SQL Server <https://www.microsoft.com/sql-server/>`_ database. This can be used to join data between
+The SQL Server connector allows querying and creating tables in an external
```

Comment:
Please revert, since the text doesn't change. Or rewrap the entire paragraph

Reply:
rewrapped complete paragraph